src/material-experimental/mdc-list/_list-theme.scss

```diff
@@ -55,6 +56,10 @@
     @include mdc-list.without-ripple($query: mdc-helpers.$mat-typography-styles-query);
     @include list-option-theme.private-list-option-typography-styles();
   }
+
+  .mat-mdc-list-item .mdc-list-item__primary-text {
```

Comment:
I'm happy with this change, but it's deviating from MDC so it would be good to know why, and also including that as a comment. What do you think?

is it just to ease some screenshots testing/the migration within g3?

Reply:
Yeah, this makes little difference etxernally, but looks much better internally. Added comment explaining